go-controller/pkg/node/iptables/iptables.go

```diff
@@ -19,57 +19,51 @@ type Rule struct {
 
 // AddRulesFiltered adds the given rules to iptables.
 // filter is a map[table][chain] of valid tables/chains to use for filtering rules to be added.
-func AddRulesFiltered(rules []Rule, append bool, filter map[string]map[string]bool) error {
+func AddRulesFiltered(rules []Rule, filter map[string]map[string]bool) error {
 	addErrors := errors.New("")
 	var err error
 	var ipt util.IPTablesHelper
-	var exists bool
 
-	// stores valid table chains and whether they were already created or not
-	// key is ip protocol, table, chain
-	createdChains := map[iptables.Protocol]map[string]map[string]bool{
```

Comment:
you don't care about chains here anymore since each chain is getting completely restored...

Reply:
right